cps/tasks/download.py

```diff
@@ -95,6 +95,12 @@ def run(self, worker_thread):
                             if error:
                                 log.error("[xklb] An error occurred while trying to download %s: %s", error[1], error[0])
                                 self.message = f"{error[1]} failed to download: {error[0]}"
+                            else:
+                                log.error("No error found in the database, likely the video failed due to unavailable fragments.")
+                                self.message = f"{self.media_url_link} failed to download: No path found in the database"
```

Comment:
@deldesir: Is this better? Can line 99 (log.error message) be strengthened similarly?

```suggestion
                                self.message = f"{self.media_url_link} failed to download: No path or error found in the database (likely the video failed due to unavailable fragments?)"
```